docs/src/openapi-v1.yaml

```
@@ -367,6 +367,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /accounts/{accountId}/proxy-info:
+    get:
+      tags:
+      - accounts
+      summary: Get proxy account information.
+      description: Returns information about a proxy account. This will include 
+        delegated accounts, and deposits held.
```

Just a small typo here:
```suggestion
        delegated accounts and deposits held.
```

---

fixed